src/main/java/am/rockstars/dto/GetProductDetailedInfoResponse.java

```diff
@@ -0,0 +1,53 @@
+package am.rockstars.dto;
+
+import am.rockstars.enums.ProductCategory;
+import am.rockstars.enums.ProductStatus;
+import am.rockstars.enums.ProductType;
+import lombok.Getter;
+import lombok.Setter;
+import lombok.ToString;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+@Getter
+@Setter
+@ToString
+public class GetProductDetailedInfoResponse {
+
+    private Long id;
+
+    private String name;
+
+    private String brand;
+
+    private String category;
+
+    private ProductCategory categoryType;
```

Comment:
As I see in JSON name should be category_type and enum value needs to be low letters